default/python/AI/AIstate.py

```diff
@@ -467,14 +483,27 @@ def update_system_status(self):
             mob_rating = CombatRatingsAI.combine_ratings_list(mob_ratings)
             if fleetsLostBySystem.get(sys_id, []):
                 lost_fleet_rating = CombatRatingsAI.combine_ratings_list(fleetsLostBySystem[sys_id])
-            if not system or partial_vis_turn == -9999:  # under current visibility rules should not be possible to have any losses or other info here, but just in case...
+            # under current visibility rules should not be possible to have any losses or other info here,
+            # but just in case...
+            if not system or partial_vis_turn == -9999:
                 if verbose:
-                    print "Have never had partial vis for system %d ( %s ) -- basing threat assessment on old info and lost ships" % (sys_id, sys_status.get('name', "name unknown"))
+                    print ("Have never had partial vis for system %d ( %s ) "
+                           "-- basing threat assessment on old info and lost ships") % (
+                        sys_id, sys_status.get('name', "name unknown"))
                 sys_status.setdefault('local_fleet_threats', set())
                 sys_status['planetThreat'] = 0
-                sys_status['fleetThreat'] = int(max(CombatRatingsAI.combine_ratings(enemy_rating, mob_rating), 0.98 * sys_status.get('fleetThreat', 0), 1.1 * lost_fleet_rating - monster_rating))
-                sys_status['monsterThreat'] = int(max(monster_rating, 0.98 * sys_status.get('monsterThreat', 0), 1.1 * lost_fleet_rating - enemy_rating - mob_rating))
-                sys_status['enemy_threat'] = int(max(enemy_rating, 0.98 * sys_status.get('enemy_threat', 0), 1.1 * lost_fleet_rating - monster_rating - mob_rating))
+                sys_status['fleetThreat'] = int(max(
+                    CombatRatingsAI.combine_ratings(enemy_rating, mob_rating),
+                    0.98 * sys_status.get('fleetThreat', 0),
+                    1.1 * lost_fleet_rating - monster_rating)
+                )
```

Comment:
I agree with that approach but this is not what you did here I think. The first argument of the ```max``` call is in the next line but the closing ) is in the same line with the arguments...

Regardless, is there any reason for all the ```int``` calls anyway? @Dilvish-fo ? It seems to me it serves no real purpose (anymore)... In that case, we should drop it as it makes the code more complicated for no actual benefit...